src/components/fields/SQFormDatePicker/SQFormDatePicker.tsx

```diff
@@ -46,6 +46,44 @@ export type SQFormDatePickerProps = BaseFieldProps & {
   isCalendarOnly?: boolean;
 };
 
+const sqFormDatePickerStyles = {
+  '& .MuiInputBase-root, & .MuiInputBase-root.Mui-focused, & .MuiInputBase-root:hover:not(Mui-disabled)':
+    {
+      '& .MuiButtonBase-root.MuiIconButton-root': {
+        color: 'var(--color-darkTeal)',
+      },
+    },
+  '& .MuiInputBase-input': {
+    fontSize: '14px',
+    fontWeight: 'var(--font-weight-semibold)',
+  },
+  '& .MuiFormHelperText-root.Mui-error': {
+    '& .MuiSvgIcon-root.MuiSvgIcon-colorError': {
+      color: 'var(--color-textWarningYellow)',
+    },
+    '& .MuiTypography-root.MuiTypography-body1': {
+      color: 'var(--color-textWarningYellow)',
+    },
+  },
+  '& .MuiInputLabel-root.Mui-error': {
+    color: 'var(--color-textWarningYellow)',
+  },
+  '& .MuiFormHelperText-root': {
+    '& .MuiSvgIcon-root': {
+      color: 'var(--color-textSuccessGreen)',
+    },
+  },
+  '& .MuiInput-root:before': {
+    borderColor: 'var(--color-stone)',
+  },
+  '& .MuiInput-root.Mui-error:before': {
+    borderBottomColor: 'var(--color-textWarningYellow)',
+  },
+  '& .MuiInput-root.Mui-error:after': {
+    borderBottomColor: 'var(--color-textWarningYellow)',
+  },
+};
+
```

Comment:
It seems like most of the overrides really should be on the theme rather than an individual component.